RELEASING.md

```diff
@@ -37,61 +48,86 @@ Label PRs with the next major version label (e.g. `3.0.0`) and merge changes int
+
 ## Versioning
 
-OpenSearch versioning [follows semver](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/). 
+All distributions in this organization [follow semver](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/). A user-facing breaking change can only be made in a major release. Any regression that broke semver is considered a high severity bug. 
 
 ### Version Numbers
 
+#### OpenSearch and OpenSearch Dashboards Version Numbers
+
 The build number of the engine is 3-digit `major.minor.patch` (e.g. `2.9.0`), while plugins use 4 digits (`2.9.0.45`). See [OpenSearch#1093](https://github.com/opensearch-project/OpenSearch/issues/1093) for a proposal to remove this difference. 
 
+#### Single Repo Version Numbers
+
+The build number of all releasable artifacts is 3-digit `major.minor.patch` (e.g. `2.9.0`).
+
 ### Incrementing Versions
 
 Versions are incremented as soon as development starts on a given version to avoid confusion. In the examples above versions are as follows.
 
 * OpenSearch: `main` = 3.0.0, `2.x` = 2.10.0 `1.x` = 1.4.0, and `1.2` = 1.2.5
 * job-scheduler: `main` = 3.0.0.0, `2.x` = 2.10.0.0 , `1.x` = 1.4.0.0 and `1.2` = 1.2.5.0
 
-## Tagging
+## Releases
 
-Create tags after a release that match the version number, `major.minor.patch`, without a `v` prefix.
+### Major Releases
 
-* [OpenSearch tags](https://github.com/opensearch-project/OpenSearch/tags): [1.0.0](https://github.com/opensearch-project/OpenSearch/releases/tag/1.0.0)
-* [job-scheduler tags](https://github.com/opensearch-project/job-scheduler/tags): [1.0.0.0](https://github.com/opensearch-project/job-scheduler/releases/tag/1.0.0.0)
+Major releases contain breaking changes, new features, and bug fixes. All artifacts released in this organization only do major releases when there are significant breaking changes ready for release that warrant a major upgrade. To avoid major upgrades we strive to build features in a backwards compatible way as much as possible, therefore it's common for a distribution to go multiple years without a major release.
 
-For a discussion on whether to add a prefixing `v` to release tags, see [#35](https://github.com/opensearch-project/.github/issues/35).  
+OpenSearch releases new major versions only when there are a critical mass of breaking changes (e.g. changes that are incompatible with existing APIs). These tend to be tied to Lucene major version releases, and will be announced in the forums at least 4 weeks prior to the release date. Once we become aware of the need for a major version, we will start a major version release window which will be similar to a minor release, except for two things: 1) participation is mandatory for all components, and 2) the release window will be at least 4 weeks long to accommodate the testing required. 
```

Comment:
let's call out the limitation of this appraoch, right now the distribution release bundled OpenSearch, OpenSearch Dashboards and plugin. In the case OpenSearch doesn't has major change, OSD has break change, it is not clear if we should let OpenSearch Dashboards hold the critical feature(with break) to release and wait for OpenSearch. Especially the break change is caused by Security issue like CVE.

Reply:
I changed it to an example.

_For example, the OpenSearch bundle that contains OpenSearch and OpenSearch Dashboards releases new major versions after Lucene major version releases. This is announced in the forums at least 4 weeks prior to the release date. Once we become aware of the need for a major version, we start a major version release window which is similar to a minor release, except for two things: 1) participation is mandatory for all components, and 2) the release window is at least 4 weeks long to accommodate the testing required._